cellprofiler/__main__.py

```diff
@@ -1,4 +1,14 @@
 from __future__ import print_function
+
+import os
+import os.path
```

Comment:
It looks like `os.path` isn't used, this can probably be removed

Reply:
You're right, that's a leftover from a previous attempt to fix this.  Previously both os and os.path WERE set though- do we want to maintain that (and move os.path down into the rest of the imports) or do we want to just import os?

Reply:
Oh oops I didn't realize it was used below! No I'd leave it up here then.

Reply:
Why? importing `os` gives one access to `os.path`.

Reply:
@jni Unfortunately a lot of tools don't pick up on relative imports without explicit declaration. 🤷‍♂️